lib/fluent/plugin/in_sqs.rb

```diff
@@ -14,8 +14,9 @@ def initialize
     config_param :tag, :string
     config_param :sqs_endpoint, :string, :default => 'sqs.ap-northeast-1.amazonaws.com'
     config_param :sqs_url, :string
-    config_param :receive_interval, :time, :default => 1
-    config_param :max_number_of_messages, :integer, :default => 1
+    config_param :receive_interval, :time, :default => 0.1
```

Comment:
change `receive_interval` value  in specfile.
https://github.com/ixixi/fluent-plugin-sqs/blob/master/spec/lib/fluent/plugin/in_sqs_spec.rb#L44
```
    context do
      subject {instance.receive_interval}
      it{should == 1}
    end
```